ql/instruments/bonds/fixedratebond.hpp

```diff
@@ -58,7 +58,8 @@ namespace QuantLib {
                       const Period& exCouponPeriod = Period(),
                       const Calendar& exCouponCalendar = Calendar(),
                       BusinessDayConvention exCouponConvention = Unadjusted,
-                      bool exCouponEndOfMonth = false);
+                      bool exCouponEndOfMonth = false,
+                      const DayCounter& stubDayCounter = DayCounter());
```

Comment:
This might be called `firstPeriodDayCounter` to match the name in `FixedRateLeg`.  What do you think?

Reply:
That makes sense to me! Will push shortly.